src/tracking.js

```diff
@@ -103,41 +103,46 @@ export function setupLogger() {
   const userAction = getCommit()
     ? FPTI_USER_ACTION.COMMIT
     : FPTI_USER_ACTION.CONTINUE;
+  const { lang, country } = getLocale();
+  const mID = getMerchantID();
 
   sdkInitTime = Date.now();
 
-  logger.addPayloadBuilder(() => {
-    return {
-      referer: window.location.host,
-      uid: getSessionID(),
-      env: getEnv(),
-    };
-  });
-
-  logger.addTrackingBuilder(() => {
-    const { lang, country } = getLocale();
-    const mID = getMerchantID();
-
-    return {
-      [FPTI_KEY.CLIENT_ID]: getClientID(),
-      [FPTI_KEY.CONTEXT_CORRID]: getCorrelationID(),
-      [FPTI_KEY.DATA_SOURCE]: FPTI_DATA_SOURCE.PAYMENTS_SDK,
-      [FPTI_KEY.FEED]: FPTI_FEED.PAYMENTS_SDK,
-      [FPTI_KEY.INTEGRATION_IDENTIFIER]: getClientID(),
-      [FPTI_KEY.JS_SDK_LIBRARY]: jsSdkLibrary,
-      [FPTI_KEY.LOCALE]: `${lang}_${country}`,
-      [FPTI_KEY.PAGE_TYPE]: pageType,
-      [FPTI_KEY.PARTNER_ATTRIBUTION_ID]: getPartnerAttributionID(),
-      [FPTI_KEY.REFERER]: window.location.host,
-      [FPTI_KEY.SDK_INTEGRATION_SOURCE]: integrationSource,
-      [FPTI_KEY.SDK_NAME]: FPTI_SDK_NAME.PAYMENTS_SDK,
-      [FPTI_KEY.SDK_VERSION]: version,
-      [FPTI_KEY.SELLER_ID]: mID && mID.toString(),
-      [FPTI_KEY.SESSION_UID]: getSessionID(),
-      [FPTI_KEY.USER_ACTION]: userAction,
-      [FPTI_KEY.USER_AGENT]: window.navigator && window.navigator.userAgent,
-    };
-  });
+  logger.addPayloadBuilder(() => ({
+    referer: window.location.host,
+    uid: getSessionID(),
+    env: getEnv(),
+    clientId: getClientID(),
+    csnwCorrelationId: getCorrelationID(),
+    jsSdkLibrary,
+    locale: `${lang}_${country}`,
+    referrer: window.location.host,
```

Comment:
would this include the full url of the payer view invoice?

i.e.: https://www.paypal.com/invoice/payerView/details/INV2-S8L9-LN49-9RMH-5VDH